src/main/java/com/kodilla/ecommercee/domain/users/UserDto.java

```diff
@@ -13,4 +18,6 @@
     private Long id;
     private String username;
     private boolean blocked;
+    private CartDto cart;
+    private List<OrderDto> orders = Collections.emptyList();
```

Comment:
jeśli tak zainicjujesz listę to nie będziesz mógł nic do niej dodać. w sumie nie widzę żadnej wartości dodanej w takiej automatycznej inicjacji listy

Reply:
Poprawione